From 6befb9582293b2d0c46bffb93623ecc155f97bac Mon Sep 17 00:00:00 2001
From: Roqqit <roqqit@proton.me>
Date: Mon, 22 Sep 2025 18:46:41 +0000
Subject: [PATCH] Cleanup unused auxpow block in test

This block is unused in the test and the valid block case is already
checked at the end of the test.

(cherry picked from commit b88bd48eabae9137fdb2b1ee091ac1137943f67e)

diff --git a/src/test/dogecoin_auxpow_block_tests.cpp b/src/test/dogecoin_auxpow_block_tests.cpp
index 9c022bf644d744432a3e2209e01c6bbfcaf722c4..fa5ab43cee8adcfa7337d5f585cc4075be263ca0 100644
--- a/src/test/dogecoin_auxpow_block_tests.cpp
+++ b/src/test/dogecoin_auxpow_block_tests.cpp
@@ -20,10 +20,6 @@ BOOST_AUTO_TEST_CASE(mine_auxpow_blocks_test) {
     const Consensus::Params &consensus = config.GetChainParams().GetConsensus();
     const BlockValidationOptions blockopts(config);
 
-    CBlock block =
-        CreateAuxPowBlock({}, CScript() << OP_1, 0x63, 0x12345678, {uint256()},
-                          {uint256(), uint256()}, chainman.ActiveChainstate());
-
     // Block must have the auxpow version bit set if it has auxpow
     {
         CBlock block = CreateAuxPowBlock(
